gluonnlp/model/language_model.py

```diff
@@ -476,7 +535,68 @@ def standard_lstm_lm_1500(dataset_name=None, vocab=None, pretrained=False, ctx=c
     assert all((k not in kwargs or k in mutable_args) for k in predefined_args), \
            'Cannot override predefined model settings.'
     predefined_args.update(kwargs)
-    return _get_rnn_model(StandardRNN, 'standard_lstm_lm_1500', dataset_name, vocab, pretrained,
+    return _get_rnn_model(StandardRNN, 'standard_lstm_lm_1500',
+                          dataset_name, vocab, pretrained, ctx, root, **predefined_args)
+
+def user_pretrained_lm(predefined_args=None, model_name=None,
+                       dataset_name=None, vocab=None, pretrained=False, ctx=cpu(),
+                       root=os.path.join('~', '.mxnet', 'models')):
+    r"""user pretrained language model setting either in StandardRNN or AWDRNN.
```

Comment:
remove. `gluonnlp.model.get_model` already does this

Reply:
This is for using user pretrained model, not the pretrained model we provided. We need provide the api for the user.

Reply:
Couldn't this already be done using get_model(pretrained=False) and then load_params?